[BUG] Databricks 301 shim compilation error

Spark301dbShims is failing to compile with:
```
15:21:43  [INFO] Compiling 11 Scala sources to /home/ubuntu/spark-rapids/shims/spark301db/target/classes ...
15:21:47  [ERROR] [Error] /home/ubuntu/spark-rapids/shims/spark301db/src/main/scala/com/nvidia/spark/rapids/shims/spark301db/Spark301dbShims.scala:125: type mismatch;
15:21:47   found   : scala.collection.immutable.Map[String,com.nvidia.spark.rapids.TypeSig]
15:21:47   required: com.nvidia.spark.rapids.TypeSig
15:21:47  [ERROR] [Error] /home/ubuntu/spark-rapids/shims/spark301db/src/main/scala/com/nvidia/spark/rapids/shims/spark301db/Spark301dbShims.scala:126: type mismatch;
15:21:47   found   : com.nvidia.spark.rapids.TypeSig
15:21:47   required: Map[String,com.nvidia.spark.rapids.InputCheck]
15:21:47  [ERROR] two errors found
```

--- a/shims/spark301db/src/main/scala/com/nvidia/spark/rapids/shims/spark301db/Spark301dbShims.scala
+++ b/shims/spark301db/src/main/scala/com/nvidia/spark/rapids/shims/spark301db/Spark301dbShims.scala
@@ -122,9 +122,12 @@ class Spark301dbShims extends Spark301Shims {
         ExecChecks(
           (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64 +
             TypeSig.STRUCT + TypeSig.ARRAY + TypeSig.MAP).nested(),
-          Map("partitionSpec" -> (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64)),
-          TypeSig.all),
-        (runningWindowFunctionExec, conf, p, r) => new GpuRunningWindowExecMeta(runningWindowFunctionExec, conf, p, r)
+          TypeSig.all,
+          Map("partitionSpec" ->
+              InputCheck(TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64,
+                TypeSig.all))),
+        (runningWindowFunctionExec, conf, p, r) =>
+          new GpuRunningWindowExecMeta(runningWindowFunctionExec, conf, p, r)
       ),
       GpuOverrides.exec[FileSourceScanExec](
         "Reading data from files, often from Hive tables",

--- a/shims/spark311db/src/main/scala/com/nvidia/spark/rapids/shims/spark311db/Spark311dbShims.scala
+++ b/shims/spark311db/src/main/scala/com/nvidia/spark/rapids/shims/spark311db/Spark311dbShims.scala
@@ -125,9 +125,12 @@ class Spark311dbShims extends Spark311Shims {
         ExecChecks(
           (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64 +
             TypeSig.STRUCT + TypeSig.ARRAY + TypeSig.MAP).nested(),
-          Map("partitionSpec" -> (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64)),
-          TypeSig.all),
-          (runningWindowFunctionExec, conf, p, r) => new GpuRunningWindowExecMeta(runningWindowFunctionExec, conf, p, r)
+          TypeSig.all,
+          Map("partitionSpec" ->
+              InputCheck(TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64,
+                TypeSig.all))),
+          (runningWindowFunctionExec, conf, p, r) =>
+            new GpuRunningWindowExecMeta(runningWindowFunctionExec, conf, p, r)
       ),
       GpuOverrides.exec[FileSourceScanExec](
         "Reading data from files, often from Hive tables",

